documentation/modules/configuring-mtv-operator.adoc

```diff
@@ -60,12 +61,28 @@ spec:
 |`10`
 
 |`controller_filesystem_overhead`
-|Percentage of space in persistent volumes allocated as file system overhead when the `storageclass` is `filesystem`. Note, this setting can only be changed using the CLI.
+|Percentage of space in persistent volumes allocated as file system overhead when the `storageclass` is `filesystem`. `ForkliftController` CR only.
 |`10`
 
 |`controller_block_overhead`
-|Fixed amount of additional space allocated in persistent block volumes. This setting is applicable for any `storageclass` that is block-based. It can be used when data, such as encryption headers, is written to the persistent volumes in addition to the content of the virtual disk. Note, this setting can only be changed using the CLI.
+|Fixed amount of additional space allocated in persistent block volumes. This setting is applicable for any `storageclass` that is block-based. It can be used when data, such as encryption headers, is written to the persistent volumes in addition to the content of the virtual disk. `ForkliftController` CR only.
 |`0`
+
+|`vsphere_osmap_configmap_name`
+|Configuration map for vSphere source providers. This configuration map maps the operating system of the incoming VM to a {virt} preference name. This configuration map needs to be in the namespace where the {project-short} Operator is deployed (`ForkliftController` CR only).
```

Comment:
```suggestion
|Configuration map for vSphere source providers. This configuration map maps the operating system of the incoming VM to a {virt} preference name. This configuration map needs to be in the namespace where the {project-short} Operator is deployed.
```
(since you didn't add that part to `ovirt_osmap_configmap_name` below)

Reply:
@ahadas I did add it -- it's on line 84, but here I left the parentheses on, so i removed them here.